CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+### 24.1.0 [#714](https://github.com/openfisca/openfisca-core/pull/714)
+
+- Get information on a country package's entities, and their roles
+- Introduce the `/entities` endpoint for the Web API
+```json
+{
+            "description": "[...]",
```

Comment:
Could we make up a simple description? Examples will real values are always easier to grasp 🙂

Reply:
Ok, in my previous comment I thought description was the label, now I understand better why you didn't want to write the full `Household` doc. 
But I still think a one or two-sentence max doc would hep :)